docs/source/contributing.rst

```diff
@@ -124,7 +124,7 @@ Setting up pre-commit
 ~~~~~~~~~~~~~~~~~~~~~
 
 We use `pre-commit <https://pre-commit.com/>`_ to manage code linting and style.
-To set up pre-commit after activating your conda environment, run:
+To set up pre-commit **after activating** your conda environment, run:
```

Comment:
This is suggested, but not actually required - pre-commit established a git hook which will run in any environment. The only real precondition is that `pre-commit` (the command) is installed, which is easily done via conda or many other methods.